pkgs/development/interpreters/gauche/default.nix

```diff
@@ -1,20 +1,25 @@
-{ stdenv, fetchurl, pkgconfig, texinfo, libiconv, gdbm, openssl, zlib
-, mbedtls, cacert
-}:
+{ stdenv, lib, fetchFromGitHub, autoreconfHook, gaucheBootstrap, pkg-config, texinfo
+,  libiconv, gdbm, openssl, zlib, mbedtls, cacert }:
 
 stdenv.mkDerivation rec {
   pname = "gauche";
-  version = "0.9.9";
+  version = "0.9.10";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/gauche/Gauche-${version}.tgz";
-    sha256 = "1yzpszhw52vkpr65r5d4khf3489mnnvnw58dd2wsvvx7499k5aac";
+  src = fetchFromGitHub {
+    owner = "shirok";
+    repo = pname;
+    rev = "release${lib.replaceChars [ "." ] [ "_" ] version}";
+    sha256 = "1lnfhywj3p3708qxsjqr6qp0dw68fx5lvj1djbsrd213w59p1v51";
```

Comment:
This checksum is incorrect.
```
    sha256 = "0ki1w7sa10ivmg51sqjskby0gsznb0d3738nz80x589033km5hmb";
```

Reply:
This hash was obtained from `nix-prefetch-github`.

```
$ nix-prefetch-github shirok gauche --rev release0_9_10
{
    "owner": "shirok",
    "repo": "gauche",
    "rev": "d028d2e291957b066572aae4a76dbd7a75a528d7",
    "sha256": "1lnfhywj3p3708qxsjqr6qp0dw68fx5lvj1djbsrd213w59p1v51",
    "fetchSubmodules": true
}
```

Reply:
Ah I was checking the hash with `--no-fetch-submodules`. Looks good!